backend/docs/admin.md

```diff
@@ -0,0 +1,46 @@
+# <u>Backend Documentation</u>
+</br>
+
+### Admin module
+The Admin module is responsible for all the functions related to admin , for example : fetching particular admin data from the database , creating a new account , changing password etc. 
+Before executing every route function validation middleware gets executed which checks whether the input is in the given format or not . For this validation process a library called joi is used. 
+### API description
+- <code> <b> /admin </b> </code>
+    - method : <b>GET</b>
+    - response body : 
+        | Parameter   | 
```

Comment:
Can we please have a single line in `response` section of each api describing its functionality for example here we can write `Get the list of all admins`

Reply:
@Kajol-Kumari  yes sure !!

Reply:
@Kajol-Kumari  the part you mentioned is from admin documentation so should i remove admin doc from this branch as i have already created another PR for that . I will remove admin doc from this branch as this is just for auth doc . Is this fine?

Reply:
@Kajol-Kumari Can you check PR #323 . That is for admin documentation . I added those lines too.